extensions/PhabBugz/lib/Feed.pm

```diff
@@ -644,6 +644,7 @@ sub process_revision_change {
         'abandoned'       => '-',
         'accepted'        => '+',
         'changes-planned' => '?',
+        'draft'           => '?',
```

Comment:
Could we set the flag to be cleared when the state is `draft`? We should probably also do so for `changes-planned`.

I should note that I copied this mapping from [here](https://github.com/mozilla-bteam/bmo/blob/master/extensions/PhabBugz/lib/WebService.pm#L151) where the draft status isn't present - is that a bug?

Reply:
> Could we set the flag to be cleared when the state is `draft`? We should probably also do so for `changes-planned`.
> 

Yes we would just set the flag status to 'X' if it already exists, otherwise if the revision is new and in the draft state, then we would just not set a flag at all.

> I should note that I copied this mapping from [here](https://github.com/mozilla-bteam/bmo/blob/master/extensions/PhabBugz/lib/WebService.pm?rgh-link-date=2023-05-11T15%3A27%3A03Z#L151) where the draft status also isn't present - is that a bug?

Not reall but we can add it there for consistency. That mapping is for human readable version of the statuses. If the mapping did not have an entry for the state it would just show the raw status from Phabricator. In this case it would just show 'draft'. If we add a new entry it would just be 'Draft' ;)